docs/_posts/2023-08-15-keep-your-secrets.md

```diff
@@ -10,13 +10,21 @@
 # */
 
 title: Keep Your Secrets… Secret
-layout: post
-prev_url: /
+layout: blog
+kind: VSecMBlog
+prev_url: /docs/blog/
 permalink: /blog/keep-your-secrets/
+next_url: /docs/blog/
+author: Volkan Özçelik
+author_link: https://github.com/v0lkan
+published_at: "August 15, 2023"
+excerpt: In the ever-evolving landscape of cloud-native applications, secrets 
```

Comment:
Posts need to have an excerpt.

That excerpt is parsed and displayed on the main page.